config/app.php

```diff
@@ -91,7 +91,7 @@
 	|
 	*/
 
-	'locale' => 'en',
+	'locale' => env('APP_LOCALE', 'en'),
```

Comment:
In generally, I agree to exposing this and the other configuration settings via the `.env` file. But two aspects:

 1. I am not sure if Lychee honors this setting at all. Lychee uses its own home-brewed localization whose setting is stored in the DB. I don't mind to expose these settings nonetheless, but they will probably only affect some error messages and Blade templates from the framework. I only wanted to make you aware of this.
 2. If you add new environment variables, you should also update the example `.env` file. Otherwise nobody will notice that these variables exist.